### ISSUE TITLE
There should be a protection against multiple simultaneous runs 

### ISSUE THREAD

**Issue by Riduidel** [rcarmo/rss2imap#19]
I have  added execution of rss2imap in the crontab of my nas (a linksys dns-323). 
Unfortunately  due to the large number of feeds I watch, even a delay of 1 hour is not always enough to prevent multiple runs and, as a consequence, multiple copies of the same message. 
Could if be possible to have a system preventing that (I have tried using pid file in `r2e` shell script with no success)?

**Comment by rcarmo** [comment]
Yeah, I suppose something could be done about that… I don’t run this anymore, but when I did I used a wrapper for it. I’ll see if I can find it.

**Comment by Riduidel** [comment]
I think there is another way : if there was an endless run mode, I wouldn't have to cron rss2imap, and multiple instances wouldn't stack up. So, what do you think of adding that mode to the main method?

**Comment by sdt** [comment]
Do you have `pgrep` on your nas @Riduidel 

Would something like this work in your crontab?

`pgrep r2e > /dev/null || $your_existing_r2e_command`
